### PR TITLE
VM build script

### PR DESCRIPTION
**Pull request by pdvyas**

### CODE REVIEW

File: vm/ansible/roles/mariadb/tasks/centos.yml~ [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (anandpdoshi) [inline vm/ansible/roles/mariadb/tasks/centos.yml~]: looks like a bad file with extension `yml~`?

File: vm/ansible/playbook.yml [hunk 2]
```diff
@@ -0,0 +1,160 @@
+---
```
Comment (anandpdoshi) [inline vm/ansible/playbook.yml]: why 2 files? `base.yml` and `playbook.yml`? both look quite similar.

**Comment by anandpdoshi** [comment]
@pdvyas :+1:

File: vm/ansible/playbook.yml (hunk 2 above)
Comment (pdvyas) [inline vm/ansible/playbook.yml]: Gotta remove `playbook.yml`

**Pull request merged**